examples/NSW_community_gaming/tests/test_nsw_social_housie.catala_en

```diff
@@ -35,13 +35,13 @@ You could have written : "condition",
 or "content"
 
 Error token:
-┌─⯈ examples/NSW_community_gaming/tests/test_nsw_social_housie.catala_en:11.21-11.26:
+┌─⯈ test_nsw_social_housie.catala_en:11.21-11.26:
```

Comment:
this should not have changed

Reply:
Ah that was done on purpose in edd6dce0d8183212488f3a27d0558fabd598c5d2 ;
my motivation was that it's annoying for the test results to fail depending on where you were running the test from. (e.g. `clerk runtest examples/.../test.catala_en` would work but `cd examples/.../ && clerk runtest test.catala_en` would fail)

Reply:
ok but the code doing this in `clerk` should have a comment as to why we `cd` into the inline test directory.